### ISSUE TITLE
`pyreverse` does not display return types of methods in `mmd` format

### ISSUE THREAD

**Issue by DudeNr33** [PyCQA/pylint#6467]
### Bug description

`pyreverse` uses the wrong syntax for the return type of methods in MermaidJS format.
Current output for the following file:
```python
class Dummy:
    def foo(self) -> None:
        return
```
produces the following output:
```
classDiagram
  class Dummy {
    foo() -> None
  }
```
which looks like this, and the return type is missing.
```mermaid
classDiagram
  class Dummy {
    foo() -> None
  }
```

### Command used

```shell
pyreverse a.py
```

### Pylint output

```shell
see above
```

### Expected behavior

The correct syntax for mermaid would be to just use `foo() None`:
```
classDiagram
  class Dummy {
    foo() None
  }
```

```mermaid
classDiagram
  class Dummy {
    foo() None
  }
```

### Pylint version

```shell
Current main
```
